src/components/m-table-edit-field.js

```diff
@@ -49,6 +49,7 @@ class MTableEditField extends React.Component {
       <MuiPickersUtilsProvider utils={DateFnsUtils}>
         <DatePicker
           {...this.getProps()}
+          {...this.props.localization}
```

Comment:
Thanks,
Can you `{...this.props.localization}` move to `this.getProps()` inside